tensorflow/inference/docker/1.15/py3/Dockerfile.neuron

```diff
@@ -39,8 +39,6 @@ ENV MODEL_BASE_PATH=/models
 # The only required piece is the model name in order to differentiate endpoints
 ENV MODEL_NAME=model
 
-ARG OPENSSL_VERSION=1.1.1g
-
```

Comment:
Please revert changes in this  file. Let's constrain all Neuron dockerfile related changes to https://github.com/aws/deep-learning-containers/pull/864

Reply:
done